Library/Formula/seafile.rb

```diff
@@ -0,0 +1,73 @@
+class Seafile < Formula
+  homepage "http://www.seafile.com/"
+  url "https://github.com/haiwen/seafile/archive/v4.2.3.tar.gz"
+  sha256 "9f4ffd912312e7154cecb8451612674e32400b345bb50345cb257c3da8218f44"
+
+  head "https://github.com/haiwen/seafile.git"
+
+  # [FIX] fix openssl build
```

Comment:
Can you please submit these patches to the upstream developers of this project (if you haven't already) and add a link to the upstream pull-request and explanation of why the patch is needed in a comment in the formula file. Thanks!